src/config/italiaConfig.js

```diff
@@ -274,6 +274,11 @@ export default function applyConfig(voltoConfig) {
     },
     videoAllowExternalsDefault: false,
     showTrasparenzaFields: false,
+    nonContentRoutes: [
+      ...config.settings.nonContentRoutes,
+      '/login-operatore',
+      '/login',
```

Comment:
`/login` non c'è già in questa lista?

Reply:
tolte, visto anche che /login-operatore non esiste più